apps/electron/layers/main/src/__tests__/integration.spec.ts

```diff
@@ -92,8 +92,12 @@ function compareBuffer(a: Uint8Array | null, b: Uint8Array | null) {
 const electronModule = {
   app: {
     getPath: (name: string) => {
-      assert(name === 'sessionData');
-      return SESSION_DATA_PATH;
+      if (name === 'sessionData') {
+        return SESSION_DATA_PATH;
+      } else if (name === 'documents') {
+        return DOCUMENTS_PATH;
```

Comment:
What are these two names? how you get this

Reply:
Please see this https://www.electronjs.org/docs/latest/api/app#appgetpathname, there are some predefined paths that given by electron